tests/cypress/e2e/login_page/login_button.cy.js

```diff
@@ -0,0 +1,19 @@
+import {goToLogin} from '../tools/navigation/nav.js'
+import {login, loginForUser} from '../tools/login/login.js'
+
+describe('Login button test', () => {
+  it('passes', () => {
+    goToLogin();
+
+    // Tests login button type and visibility
+    cy.get('button[class*="green-btn mt-2"]')
```

Comment:
Not sure why we get the button on the classes. What if we decide to re-style the login button? The e2e tests would fail in this case! However, the functionality would still be the same.

I assume we can also get the button by the id or value? I know that both are not set at this moment, but feel free to modify the templates as well to remove ambiguity for testing. It would make sense to simply give this button an id to clarify the template as well as the tests (and remove the dependency on classes):

`<button type="submit" class="green-btn mt-2" id="login_button">{{_('login')}}</button>`

Reply:
Ah, I didn't know we would be allowed to change anything outside of our test, but this helps a lot!